docs/_optimist/faq/index.de.md

```diff
@@ -110,3 +110,29 @@ UUID zu verwenden. Zum Beispiel:
 
 Wir unterstützen keine multi-attached Volumes in der Optimist Platform, da für die Nutzung von multi-attached Volumes clusterfähige Dateisysteme erforderlich sind, um den gleichzeitigen Zugriff auf das Dateisystem zu koordinieren.
 Versuche, mutli-attached Volumes ohne clusterfähige Dateisysteme zu verwenden, bergen ein hohes Risiko der Datenkorruption, daher ist diese Funktion auf der Optimist Plattform nicht aktiviert.
+
+## Warum kann ich keinen Snapshot einer laufenden Instance erstellen?
+
+Um konsistente Snapshots bereitzustellen, verwendet die Optimist-Plattform die Property [os_require_quiesce=yes](https://opendev.org/openstack/nova/commit/926e58a179ef373646164bea40dc46b1ebef4748).
+Diese Eigenschaft ermöglicht es `fsfreeze`, den Zugriff auf laufende Instanzen auszusetzen und fortzusetzen, und stellt sicher, dass ein stabiles Image auf der Festplatte beibehalten wird.
+
+Benutzer müssen daher einen der folgenden Ansätze wählen, um Snapshots auf laufenden Instanzen zu erstellen:
```

Comment:
Zur Erstellung von Snapshots von Instanzen stehen in der Optimist Platform die folgenden Optionen zur Verfügung: